po.js

```diff
@@ -175,6 +175,7 @@ PO.prototype.replaceAll = function (string, find, replace) {
 }
 
 PO.prototype.fixString = function (savedString) {
+    savedString = this.replaceAll(savedString, "\\", '\\\\');
```

Comment:
Ok. Thank you. Would be good to have such things separately in different pull requests it next time.